soccer/gameplay/plays/DefendClear.py

```diff
@@ -0,0 +1,31 @@
+import standard_play
+import behavior
+import skills
+import main
+
+
+## Simplistic play to handle defend clear situation
+#  Mimicks basic122 offense. One pivot kick and two marking robots
+#
+class DefendClear(standard_play.StandardPlay):
+    def __init__(self, num_supports=2):
+        super().__init__(continuous=False)
+
+        self.add_transition(behavior.Behavior.State.start,
+                            behavior.Behavior.State.running, lambda: True,
+                            'Immediately transition to running')
+
+        best_opps_to_mark = self.closest_opps_to_our_goal(num_supports)
+        self.add_subbehavior(skills.pivot_kick.PivotKick(), 'main')
+        for i in range(num_supports):
+            mark_skill = skills.mark.Mark()
+            mark_skill.mark_robot = best_opps_to_mark[i]
+            self.add_subbehavior(mark_skill, 'mark' + str(i))
+
+    ## Returns the robots that are closest to our goal
+    #
```

Comment:
Are you making sure to add your score function?

Reply:
Chris said to leave it blank because we'll be working on it later

Reply:
Just kidding. You were right; we needed to add it